bpf/lib/l3.h

```diff
@@ -144,8 +147,11 @@ static __always_inline int ipv6_local_delivery(struct __ctx_buff *ctx, int l3_of
 static __always_inline int ipv4_local_delivery(struct __ctx_buff *ctx, int l3_off,
 					       __u32 seclabel, struct iphdr *ip4,
 					       const struct endpoint_info *ep,
-					       __u8 direction, bool from_host,
-					       bool hairpin_flow)
+					       __u8 direction __maybe_unused,
+					       bool from_host __maybe_unused,
+					       bool hairpin_flow __maybe_unused,
+					       bool from_tunnel __maybe_unused,
+					       __u8 cluster_id __maybe_unused)
```

Comment:
Note that all the `__maybe_unused` annotations are gone, and I think the `cluster_id` won't need it either.